### ISSUE TITLE
Plugin seems to work with Neovim 0.8.2

### ISSUE THREAD

**Issue by nitnelave** [nvim-treesitter/playground#118]
Not really an issue, but it seems to work fine with this version, not requiring nightly (which can be scary for people).

Whatever you were waiting for in neovim seems to have been stabilized. It could be nice to find the minimum neovim version required rather than nightly?

**Comment by clason** [comment]
No, stabilization is still not happening -- there will be significant breaking changes in the future.

Neovim officially only supports two versions:
1. the latest release (now 0.8.3; there's no reason older point releases as these should be non-breaking)
2. the latest commit on master (and no earlier commit!), also referred to as "nightly"

Nightly is recommended since there are ongoing major improvements to treesitter.
